### PR TITLE
Directory Permissions - Changed to use constants

### PR DESCRIPTION
**Pull request by thpang67**
I have updated the code to use constants in place of the 0755 along with a note about possibly using the umask of the system git-lfs is installed.

### CODE REVIEW

**Comment by rubyist** [comment]
This looks good, but I think the comment about future use of umask should be left out. The umask should already be honored when creating files and directories via Go code. That is, we ask for `0755` but if the umask is set to something other than `0022` we won't get it.

One formatting nit-pick, I think we'd rather see related consts blocked together. Also Go code will generally camel case the names rather than use all caps. You can find counterexamples such as `os.O_RDONLY` or `os.SEEK_SET` and friends, but those are usually coming from POSIX or underlying system names. I think we'd want to camel case here, e.g.

```go
const (
	TempDirPerms       = 0755
	LocalMediaDirPerms = 0755
	LocalLogDirPerms   = 0755
)
```

Thanks for the PR!

File: lfs/lfs.go [hunk 1]
```diff
@@ -14,6 +14,15 @@ import (
...
 const Version = "0.5.1"
 
+//
+// Setup permissions for the given directories used here.
+//
+const (
+    TempDirPerms       = 0755
+    LocalMediaDirPerms = 0755
+    LocalLogDirPerms   = 0755
+)
```
Comment (michael-k) [inline lfs/lfs.go]: For now, there's no need to export these constants.

**Comment by technoweenie** [comment]
@thpang67 Thanks, just merged this.  I have some Go feedback for you, under the assumption that you're pretty new to Go in general:

We don't need to expose those values because they only matter to the internals of the `lfs` package. I made them lowercase to keep them private in 4eb1660fa7e968e9e67931827197095989e6178d.

I noticed that commit [changed the spacing on the constants](https://github.com/github/git-lfs/commit/4eb1660fa7e968e9e67931827197095989e6178d#diff-0a0453ed1c5381f97debc0f1ffa878fcL20). I use the `go-plus` atom package to run `goimports` on every Go file after a save automatically. The project will also run `script/fmt` at the very start of every `script/test` or `script/cibuild` run too.

**Comment by thpang67** [comment]
@technoweenie thanks for the information on the atom package. I have just started using atom and find it to be quite useful. I have added the go-plus package. Go is new to me. I am coming from many years of structured development and programming. Thanks again.

**Pull request merged**